### PR TITLE
Problem: travis config is out of date

### PR DESCRIPTION
**Pull request by dkliban**
Solution: update using plugin-template

[noissue]

### CODE REVIEW

**Comment by daviddavis** [comment]
Are these files being used anymore?

https://github.com/pulp/pulp_ansible/blob/master/.travis/playbook.yml
https://github.com/pulp/pulp_ansible/blob/master/.travis/ansible_playbook.yml

**Comment by dkliban** [comment]
No ...

> Are these files being used anymore?
> 
> https://github.com/pulp/pulp_ansible/blob/master/.travis/playbook.yml
> https://github.com/pulp/pulp_ansible/blob/master/.travis/ansible_playbook.yml

No, I'll remove. thank you for pointing it out.

File: .travis/install.sh [hunk 1]
```diff
@@ -7,18 +7,123 @@
...
+    # k3s local-path requires this
+    access_mode: "ReadWriteOnce"
+    # We have a little over 40GB free on Travis VMs/instances
+    size: "40Gi"
+  image: $PLUGIN
+  tag: $TAG
+  database_settings:
+    username: pulp
+    password: pulp
+    admin_password: pulp
```
Comment (mikedep333) [inline .travis/install.sh]: This got renamed in https://github.com/pulp/pulp-operator/pull/23

File: .travis/before_install.sh [hunk 2]
```diff
@@ -19,77 +19,99 @@ if [ -f $PRE_BEFORE_INSTALL ]; then
...
+# test_requirements contains tools needed for flake8, etc.
+# So install them here rather than in install.sh
 pip install -r test_requirements.txt
 
 # check the commit message
 ./.travis/check_commit.sh
 
-# run black separately from flake8 to get a diff
-black --check --diff .
+
```
Comment (daviddavis) [inline .travis/before_install.sh]: :cry:

**Pull request merged**